common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -660,4 +660,15 @@ trait FeatureSwitches {
     exposeClientSide = true,
     highImpact = false,
   )
+
+  val EnableHlsWeb = Switch(
+    group = SwitchGroup.Feature,
+    name = "enable-hls-web",
+    description = "Enable HLS web streaming on web",
+    owners = Seq(Owner.withEmail("fronts.and.curation@guardian.co.uk")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Did you intend to leave `sellByDate = never`?

The positive is you won't be prompted to revisit the switch at a later date.
The negative is you won't be prompted to revisit the switch at a later date.